Gemfile

```diff
@@ -108,7 +108,7 @@ gem "foundation-rails"
 
 gem 'jquery-migrate-rails'
 gem 'jquery-rails', '3.1.5'
-gem 'jquery-ui-rails', '~> 4.0.0'
+gem 'jquery-ui-rails', '~> 4.2'
```

Comment:
As with the other PR, shall we make it `4.2.1` here? The `Gemfile.lock` is using `4.2.1` anyway.

Reply:
We are not stuck at 4.2.1 we are only stuck below v5. I think this is the correct way to say we can take any v4 above 4.2 version but not v5.
I think an upgrade to v4.3 or above would be welcome and will most probably work.
Does this make sense to you or do you prefer 4.2.1. here?

Reply:
Ah, I see :okay_hand: